### ISSUE TITLE
focus on highlighted status

### ISSUE THREAD

**Issue by FND** [innoq/statuses#169]
within lengthy conversations, individual statuses might disappear beneath the fold - e.g. if .../updates/3148 has numerous preceding messages within the same thread, only the thread's top messages are visible on page load

given the resource design and content layout constraints, I can't think of a decent solution though (I'd rather avoid a JavaScript hack)

**Comment by aheusingfeld** [comment]
What about using the good old named anchor to "scroll" to the highlighted status?

**Comment by FND** [comment]
good point, I forgot to articulate my assumption there: I reckon it'd be weird if a status's canonical URI included a redundant anchor (.../updates/3148#3148)

**Comment by aheusingfeld** [comment]
well, I actually thought of something like `<meta http-equiv="refresh" content="1;URL=#current"/>` and adding the `id="current"` instead of `class="current"`

**Comment by FND** [comment]
yeah, I'm not sure we wanna go there - do we?
